### PR TITLE
2.5 changelog structuring

### PR DESCRIPTION
**Pull request by ronso0**
this is only a first step.

### TODO
- [ ] double-check `/res/linux/org.mixxx.Mixxx.metainfo.xml`, backticks ` in CHANGELOG.md are causing linebreaks

### CODE REVIEW

**Comment by daschuer** [comment]
Oh sorry, we touched the same file at the same time .. 
Do you want to cherry pick: https://github.com/mixxxdj/mixxx/pull/13315

I will now also prepare a 2.4 changelog update.

**Comment by ronso0** [comment]
Thanks, I picked the changes manually, not in the mood to fix pick/merge conflicts ; )

**Comment by ronso0** [comment]
Doe sany have an idea how to fix the issue with metainfo hook?
> double-check /res/linux/org.mixxx.Mixxx.metainfo.xml, backticks ` in CHANGELOG.md are causing linebreaks

File: CHANGELOG.md.backup
Comment (daschuer) [inline CHANGELOG.md.backup:1]: This is likely included unwanted.

**Comment by daschuer** [comment]
> Does any have an idea how to fix the issue with metainfo hook?

This is not an issue, because they are only in the xml source not in the rendered HTML. 
The issue is that appstream-util does not support <code> and that's why we remove the code tags. 

https://github.com/mixxxdj/mixxx/blob/4d9db3e5d0e4807bdcddaebf8c87bdc3e3970d53/tools/update_metainfo.py#L87-L89

File: CHANGELOG.md.backup
Comment (ronso0) [reply CHANGELOG.md.backup:1]: Oh, thanks.
FWIW I tried Ghostwriter this time: it has a markdown live preview, but it's slow with the 2k+ lines of CHANGELOG.md

**Comment by ronso0** [comment]
Alright, thanks for the clarification!

**Comment by daschuer** [comment]
Is this ready for merge than?

**Comment by ronso0** [comment]
yes, Only the remaining items in the Features section should be sorted somewhere else.
I'll be afk, maybe the entire day, so if anyone wants to take over or just merge and fix directly, please go ahead.

**Review by daschuer — APPROVED** [review]
I will take over. Thank you so far.

**Comment by ronso0** [comment]
Thanks, I was about to start the finish : )

**Pull request merged**